### PR TITLE
[ENHANCEMENT] Refactored password ResettingController for better inheritance

### PR DESCRIPTION
**Pull request by christianbader**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+
+    * ENHANCEMENT #1251 [SecurityBundle] Refactored password PasswordResetting controller
```
Comment (turbo-ele) [inline CHANGELOG.md]: hm?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+
```
Comment (danrot) [inline CHANGELOG.md]: No new line here

File: src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php [hunk 3]
```diff
@@ -18,23 +23,31 @@
...
+    protected static $translationDomain = 'backend';
     const ENTITY_NAME_USER = 'SuluSecurityBundle:User';
-    const EMAIL_SUBJECT_KEY = 'security.reset.mail-subject';
-    const EMAIL_MESSAGE_KEY = 'security.reset.mail-message';
     const MAX_NUMBER_EMAILS = 3;
 
     /**
+     * Returns the sender's email address.
+     *
+     * @param Request $reqeust
```
Comment (turbo-ele) [inline src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php]: request

**Comment by turbo-ele** [comment]
else, very well done christian

**Pull request merged**